package.json

```diff
@@ -156,7 +156,7 @@
       "/src/test/utils.js",
       "/src/test/gql.js",
       "/src/test/__mocks__",
-      "/src/schema/v2"
+      "src/schema/v2/__tests__/ecommerce/"
```

Comment:
Isn’t this for the fake stitching and thus shouldn’t we just get rid of them entirely?

Reply:
Probably, yeah.